a11y/test-config.yml

```diff
@@ -0,0 +1,168 @@
+branding: a11y-test
+title: a11y test environment
+homeTimezone: America/Chicago
+
+# Default OTP API
+api:
+  host: http://localhost:9999
+  # host: http://localhost:8001 # For testing against a local OTP instance
+  path: /otp/routers/default
+
+# Enabled multimodal routing types (e.g. interary, profile)
+routingTypes:
+  - key: ITINERARY
+    text: Exact Time
+
+# Show/hide elevation profile chart for walk/bike legs
+# elevationProfile: true
+
+# Map config
+map:
+  # Enabled map views
+  views:
+    - type: DEFAULT
+      text: Map View
+    # Delete/uncomment the following block to disable the stylized map
+    #- type: STYLIZED
+    #  text: Network View
+
+  # Default map center
+  initLat: 29.7604
+  initLon: -95.3698
+  initZoom: 12
+
+  # Base layers for default map
+  baseLayers:
+    - name: Streets
+      url: //api.mapbox.com/styles/v1/mapbox/outdoors-v11/tiles/{z}/{x}/{y}{retina}?access_token=pk.eyJ1IjoiaWJpLXRyYW5zaXQtZGF0YS10ZWFtIiwiYSI6ImNqeTY3Zms2bzBkbjMzbGs5bWpoemRxY3UifQ.IrkMhr7JMhUNcb2qXuzwig
+      attribution: "© <a href='https://www.mapbox.com/about/maps/'>Mapbox</a> © <a href='http://www.openstreetmap.org/copyright'>OpenStreetMap</a> <strong><a href='https://www.mapbox.com/map-feedback/' target='_blank'>Improve this map</a></strong>"
+      maxZoom: 20
+      hasRetinaSupport: true
+      retina: "@2x"
+
+  overlays:
+    - type: bike-rental
+      name: BCycle Locations
+      modes:
+        - BICYCLE_RENT
+      companies:
+        - BCYCLE
+      mapSymbols:
+        - maxZoom: 12
+          minZoom: 0
+          type: circle
+          pixels: 3
+          fillColor: "#FF2E28"
+          dockStrokeColor: "#000000"
+        - maxZoom: 15
+          minZoom: 13
+          type: circle
+          pixels: 5
+          fillColor: "#FF2E28"
+          dockStrokeColor: "#000000"
+        - maxZoom: 20
+          minZoom: 16
+          fillColor: "#FF2E28"
+          dockStrokeColor: "#000000"
+          type: hubAndFloatingBike
+
+    - type: park-and-ride
+      name: Park & Ride Locations
+      maxTransitDistance: 1000
+      modes:
+        - CAR_PARK
+
+    - type: stops
+      name: Transit Stops
+      visible: true
+
+geocoder:
+  # IBI geocode.earth proxy
+  baseUrl: https://cglx01n2ua.execute-api.us-east-1.amazonaws.com/prod
+  # Geocoding search bounds
+  boundary:
+    rect:
+      minLon: -97.4197
+      maxLon: -93.2674
+      minLat: 28.6093
+      maxLat: 30.9685
+  # fetch options
+  options:
+    headers:
+      # AWS API Gateway key specific to Houston (IBI AWS account):
+      # https://console.aws.amazon.com/apigateway/home?region=us-east-1#/api-keys/fchsdgyys9
+      x-api-key: lvJXqETokn3T338y1c8U843Fjh1ONdgzK3Ar0pl4
```

Comment:
Can these API gateway keys be populated from somewhere else and not actually live in this file?

Reply:
None of these keys are needed. Removed in b9200a6d143e74ae12e63673d6acc8a44ac88b08. Apologies.